integration-tests/tests/src/tests/list.ts

```diff
@@ -1187,11 +1189,12 @@ describe("Lists", () => {
         expect(arrayCopy[0]).equals(null);
 
         this.realm.deleteAll();
+        expect(objectsCopy.isValid()).equals(true);
```

Comment:
Perhaps a matter of taste, but `to.be.true/false` seem to be more prevalent in our tests.
```suggestion
        expect(objectsCopy.isValid()).to.be.true;
```